plugins/woocommerce/src/Admin/Features/OnboardingTasks/TaskList.php

```diff
@@ -407,7 +407,7 @@ public function get_json() {
 			'isComplete'            => $this->is_complete(),
 			'tasks'                 => $tasks_json,
 			'eventPrefix'           => $this->prefix_event( '' ),
-			'displayProgressHeader' => $this->display_progress_header,
+			'displayProgressHeader' => $this->display_progress_header && ! $this->is_complete(),
```

Comment:
I don't think we want to hide this everywhere when tasks are complete, probably just in the sidebar.

The `Welcome to {siteTitle}` feels a little weird inside the task list anyway.  I wonder if we could just move that portion outside the task list on the homepage.  cc @louwie17 for your thoughts.

If that's not an option, we could add an `is-complete`  class to the task list so we can hide it via CSS in the sidebar.

Reply:
I assumed this would apply to both the side bar as well as the Homescreen, since I wasn't sure it made sense to display the progress in general after everything was complete. But perhaps not? 

I'm not sure of all the requirements around the progress bar, but glad to make this change. At this point further changes will have to wait until Monday, though.

Reply:
I agree with @joshuatf here, I think the `displayProgressHeader` here should not be dependant on is_complete and that part being handled on the front-end instead. In conjunction with moving the `Welcome to {siteTitle}` to the home screen.

The welcome site title header was part of the progress header when the completed header wasn't insight yet. Now that we do have the completed header, we probably only want to display the `Welcome to site` header when a task list is hidden, which can be handled on the homepage.
We might want to double check that logic with @jarekmorawski